core/client/app/components/gh-tag-settings-form.js

```diff
@@ -108,16 +108,16 @@ export default Component.extend({
     },
 
     actions: {
-        setProperty(property, value) {
-            this.get('setProperty')(property, value);
+        setProperty() {
+            invokeAction(this, 'setProperty', ...arguments);
```

Comment:
I guess it was an effort to be more concise but on second thought it makes this block a bit less clear. I'll change it back 😄